src/ansys/motorcad/core/geometry.py

```diff
@@ -39,6 +40,14 @@ def __eq__(self, other):
             and self.entities == other.entities
         )
 
+    @classmethod
+    def from_coordinate_list(cls):
```

Comment:
Need to refactor geometry.py before this is possible. Think this needs doing anyway since this is getting huge